solhint.config.js

```diff
@@ -5,6 +5,7 @@ const rules = [
   'const-name-snakecase',
   'contract-name-capwords',
   'event-name-capwords',
+  'max-states-count',
```

Comment:
Why add this rule?

Reply:
Restricts the amount of state variables added to a contract. I just thought it was fine

Reply:
To 15. I assume that we would just bypass it if it starts complaining.

Reply:
we can remove it if it bothers us

Reply:
Yeah let's do it